packages/local-managed-provider/readme.md

```diff
@@ -0,0 +1,185 @@
+# Filecoin wallet provider
```

Comment:
I think we can clean this way up and only include:

- class instantiation
- `getAccounts` call
- `sign` call